src/shared/ui/modals/deleteConfirmModal/DeleteConfirmModal.tsx

```
@@ -0,0 +1,34 @@
+  onClose,
+  onConfirm,
+  title,
+  description,
+}: DeleteConfirmModalProps) {
+  return (
+    <Dialog open={open} onOpenChange={onClose}>
+      <DialogContent className="flex flex-col items-center max-w-80">
+        <DialogTitle>{title}</DialogTitle>
+        <div className="my-4 text-gray-300">{description}</div>
+        <div className="flex justify-end gap-2 mt-6">
+          <button className="px-4 py-2 rounded bg-gray-400" onClick={onClose}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)